tests/backends/ffi_import_00.c

```diff
@@ -0,0 +1,19 @@
+// FFI testing support functions to be imported into ASL test programs
+// Copyright (C) 2025-2025 Intel Corporation
+#include <stdint.h>
+
+// 8, 16, 32 and 64-bit bitvectors are represented using uint*_t
+uint32_t FFI_Invert32(uint32_t x)
+{
+        return ~x;
+}
+
+// Wide bitvectors are represented as arrays of uint64_t
+// If returning a wide bitvector, an extra array argument is
+// added as the last argument
+void FFI_Invert128(uint64_t x[2], uint64_t y[2])
```

Comment:
```suggestion
void FFI_Invert128(uint64_t x[], uint64_t y[])
```
Strictly speaking, array sizes are not needed here. I assume you add them for readability.

Reply:
Yes, I want the size of the arrays to be clear since the size of the array is important for writing correct code.
I am not sure whether gcc/clang generate warnings if you are obviously writing past the end of an array - but hopefully a human code reviewer would spot a problem.